mix.exs

```diff
@@ -72,4 +76,18 @@ defmodule PropCheck.Mixfile do
       {:credo, "~> 1.0.0", only: [:dev, :test], runtime: false}
     ]
   end
+
+  defp test_suite(), do: [
+    &run_external_tests/1,
```

Comment:
I would have expected the external tests to run last. Did you have a specific reason to place them first? (I don't have a real preference, just curious)

Reply:
Just to have coverage report and tests result line (_21 doctests, 55 properties, 17 tests, 1 failure, 23 excluded_) easily visible. I did start with the order you were expecting and then I was scrolling up, so I switched it. The reverse order needs `propcheck.clean` in between though. :confused: